forcewindow.h

```diff
@@ -41,6 +41,7 @@ private slots:
     void lostConnection();
 
 //    void setForceValue( QString );
+// QA: why not float& force_kg? Is it declared somewhere in Qt Lib? 
     void setForceValue( float force_kg );
```

Comment:
Sorry, I'm not sure, that there is a big difference between `float` and `float&`. Could you explain?